hivemind/averaging/averager.py

```diff
@@ -138,19 +121,14 @@ def __init__(
 
         super().__init__()
         self.dht = dht
-        self.client_mode, self.listen_on, self.kwargs = client_mode, listen_on, kwargs
+        self.client_mode = client_mode
```

Comment:
nit: perhaps we should set default client_mode=None

and
```python
if client_mode is None:
    client_mode = dht.client_mode
```
?

Reply:
Sounds like a good idea since it is easy to set this flag for the DHT and forget to also set it for the averager.

I've implemented this.